perun-core/src/main/java/cz/metacentrum/perun/core/entry/VosManagerEntry.java

```diff
@@ -278,6 +273,7 @@ public Vo getVoById(PerunSession sess, int id) throws VoNotExistsException, Inte
 
 		// Authorization
 		if (!AuthzResolver.isAuthorized(sess, Role.VOADMIN, vo) &&
+				!AuthzResolver.isAuthorized(sess, Role.VOOBSERVER, vo) &&
```

Comment:
Now when I think about it. VOOBSERVER can't actually add candidate as a member of VO. So there is no real reason, why he should be able to read them. Also it retrieves data from ext sources - people who are not yet in the VO (possibly perun), but the role VOOBSERVER should grant privilege to see data in Perun only.

Reply:
I think VOOBSERVER can see anything what VOADMIN can see, so there is no problem since there is no modification.

Reply:
OK.